File: CS/Office2010StyleApplicationMenu102/Form1.cs
using System;
using DevExpress.XtraBars.Ribbon;

namespace Office2010StyleApplicationMenu102
{
    public partial class Form1 : RibbonForm
    {
        public Form1()
        {
            InitializeComponent();
            ribbon.ApplicationButtonDropDownControl = new ApplicationButtonDropDownControl();
        }
    }
}

File: VB/Office2010StyleApplicationMenu102/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraBars.Ribbon

Namespace Office2010StyleApplicationMenu102
	Partial Public Class Form1
		Inherits RibbonForm
		Public Sub New()
			InitializeComponent()
			ribbon.ApplicationButtonDropDownControl = New ApplicationButtonDropDownControl()
		End Sub
	End Class
End Namespace
